README.md

```diff
@@ -6,11 +6,11 @@ These are the required headers for each request:
 
 *   "**User-Agent**" - Please identify the system/product you are using to make this request.
     
-*   "**X-Auth-Date**" - The current unix epoch time as a string. 5 minute window.
+*   "**X-Auth-Date**" - The current unix epoch time as a string. 5 minute window.  If accessing from Python, use only the integer portion of the time value.  Note that JavaScript's time value from a Date's getTime() is compatible, but needs to be scaled down by 1,000 before also being converted to an integer.
     
 *   "**X-Auth-Key**" - Your API key string.
     
-*   "**Authorization**" - A SHA-1 hash of the X-Auth-Key, the corresponding key secret and the X-Auth-Date value concatenated as a string. The resulting hash should be encoded as hexadecimal value.
+*   "**Authorization**" - An SHA-1 hash of the X-Auth-Key, the corresponding key, secret and the X-Auth-Date value concatenated as a string. The resulting hash should be encoded as a hexadecimal value, two digits per byte, using lower case letters for the hex digits "a" through "f".
```

Comment:
1) If pronounce SHA-1 like the Iranian Monarch (as I tend to) then the original grammar here is correct I believe. 
2) Adding the `a` is a good idea. 
3) I'm not sure that the final addition is necessary, I haven't run into a problem with using SHA-1 yet, is that an issue you had?